cmd/zetae2e/run.go

```diff
@@ -67,6 +74,16 @@ func runE2ETest(cmd *cobra.Command, args []string) error {
 	// initialize logger
 	logger := runner.NewLogger(verbose, color.FgHiCyan, "e2e")
 
+	// update config with dynamic ERC20
+	erc20ChainName, _ := cmd.Flags().GetString(flagERC20ChainName)
```

Comment:
The only time an error would be thrown would be if the flag wasn't defined. Which wouldn't happen since we use the shared variable to define and get it.